source/includes/_embed2.md.erb

```diff
@@ -375,6 +375,14 @@ When called, the `zdChatterAuthCallback` can be used to request a JWT token from
 ## Actions
 Actions can be called from the global adaEmbed object. A list of the available actions is available below.
 
+### closeCampaign
+`closeCampaign: Promise<void>;`
+
+Can be used to programatically close campaign.
```

Comment:
I suggest something like: "Closes the currently-displayed campaign (does nothing if no campaign is currently displayed)."